desktop/apps/artwork/components/partner_stub/index.coffee

```diff
@@ -11,9 +11,9 @@ module.exports = ($el) ->
           login: 'Log in to Artsy to call gallery'
           signup: 'Create an Artsy account to call gallery'
           register: 'Create an Artsy account to call gallery'
-        artistIds: JSON.parse(e.toElement.dataset.artist_ids)
+        artistIds: JSON.parse(e.target.dataset.artist_ids)
```

Comment:
I think you may even want `currentTarget` instead of `target` to guarantee you always get the ⚓️ element holding the data, in case someone adds more elements inside it.